indexer/services/comlink/src/controllers/api/v4/affiliates-controller.ts

```diff
@@ -0,0 +1,72 @@
+import { stats } from '@dydxprotocol-indexer/base';
+import express from 'express';
+import { checkSchema, matchedData } from 'express-validator';
+import {
+  Controller, Get, Query, Route,
+} from 'tsoa';
+
+import { getReqRateLimiter } from '../../../caches/rate-limiters';
+import config from '../../../config';
+import { handleControllerError } from '../../../lib/helpers';
+import { rateLimiterMiddleware } from '../../../lib/rate-limit';
+import ExportResponseCodeStats from '../../../request-helpers/export-response-code-stats';
+import { AffiliateReferralCodeRequest, AffiliateReferralCodeResponse } from '../../../types';
+
+const router: express.Router = express.Router();
+const controllerName: string = 'affiliates-controller';
+
+@Route('affiliates')
+class AffiliatesController extends Controller {
+  @Get('/referral_code')
+  async getReferralCode(
+    @Query() address: string,
+  ): Promise<AffiliateReferralCodeResponse> {
+
+    // simulate a delay
```

Comment:
nit: add `TODO(OTE-???): description`

Reply:
another nit: ideally follow the format, because we follow the format of `TODO(OTE-???): in all of our todos today

Reply:
Will update this in upcoming PR for other stubs